scoresheets/PickAndPlaceChallenge.tex

```diff
@@ -0,0 +1,42 @@
+\begin{scorelist}[timelimit=10]
```

Comment:
I dont think we can fit a 10 minute task into the schedule. (On the day with restaurant)

All other tasks are 7 minutes, fitting into 3h timeslots.
We can maybe just raise the scores a little to compensate? (if the task is designed for 10minutes.)